imgui_markdown.h

```diff
@@ -917,6 +876,72 @@ namespace ImGui
         return c_ != ' ' && c_ != '.' && c_ != ',' && c_ != ';' && c_ != '!' && c_ != '?' && c_ != '\"';
     }
 
+    // ImGui::TextWrapped will wrap at the starting position
+    // so to work around this we render using our own wrapping for the first line
+    inline void TextRegion::RenderTextWrapped( const char* text_, const char* text_end_, bool bIndentToHere_ )
+    {
+    #if IMGUI_VERSION_NUM >= 19197
```

Comment:
Could you indent macros the same way as any other code? This is the style used in the previous/rest of the code (this is not a PR breaking issue, I can reformat after merge).